sktime/classification/deep_learning/lstmfcn.py

```diff
@@ -132,8 +132,9 @@ def build_model(self, input_shape, n_classes, **kwargs):
             optimizer="adam",
             metrics=["accuracy"],
         )
-
-        self.callbacks = self.callbacks or []
+        # .get_params() returns an empty list for callback.
+        # inconsistent with function initial run where callbacks was set to None
+        self.callbacks = self.callbacks or None
```

Comment:
`self.callbacks` should never be changed

Reply:
If it is left to an empty list, then the error re-occur's.

Reply:
yes, please

Reply:
Got it! I will make correct changes asap!